src/amrutaM/testcases/TC10_PrintLastnameUsername.java

```diff
@@ -0,0 +1,38 @@
+package amrutaM.testcases;
+
+import java.util.ArrayList;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import amrutaM.base.PredefinedActions;
+
+/*Test case : 10
+  Name -> "Maulik", print lastName and UserName
+*/
+public class TC10_PrintLastnameUsername {
+	
+	ArrayList<String> getEmployeeInfo(WebDriver driver , String empName) {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method